docs/README.md

```diff
@@ -34,6 +34,18 @@ $ yarn build
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
+Note that this will have quite some broken links as we are referring to
+generated documentation, test data and benchmarks. To put these artifacts at the
+right place before, you can use these `nix` builds from the repository root:
```

Comment:
These instructions point to the fact we are missing different kind of documentations which should not be mixed? In the diataxis framework terminology, those elements would be [reference](https://diataxis.fr/reference/) and perhaps they should have their own dedicated (sub-)site? 

Suggestion: Leave the details out for the moment, or point to how it's done in the `publish-docs.yaml`